src/amr/data/particles/refine/split_2d.h

```diff
@@ -0,0 +1,481 @@
+        0.046875, // 1 == ORANGE
+        0.015625  // 2 == RED
+    };
+};
+
+
+/**************************************************************************/
+template<>
+class Splitter<DimConst<2>, InterpConst<2>, RefinedParticlesConst<4>, Color::PURPLE>
+    : public ASplitter<DimConst<2>, InterpConst<2>, RefinedParticlesConst<4>>
+{
+    using Dispatcher = SplitParticleDispatcher<DimConst<2>, RefinedParticlesConst<4>>;
+
+public:
+    Splitter() { split(weights_, deltas_); }
+
+    template<typename Weights, typename Deltas>
+    static void split(Weights& weights, Deltas& deltas)
+    {
+        split(weights, weight_val_, deltas, delta_val_);
+    }
+    template<typename Weights, typename Deltas>
+    static void split(Weights& weights, float const& weight_val, //
+                      Deltas& deltas, float const& delta_val)
+    {
+        Dispatcher::set_weights(weights, weight_val);
+        Dispatcher::set_deltas(deltas, delta_val);
+    }
+
+protected:
+    constexpr static float delta_val_  = 0.683734;
+    constexpr static float weight_val_ = 0.25;
+};
+
+
+template</*default pattern*/>
+class Splitter<DimConst<2>, InterpConst<2>, RefinedParticlesConst<4>>
+    : public Splitter<DimConst<2>, InterpConst<2>, RefinedParticlesConst<4>, Color::PURPLE>
+{
+};
+
+
+/**************************************************************************/
+template<>
+class Splitter<DimConst<2>, InterpConst<2>, RefinedParticlesConst<8>, Color::PINK, Color::PURPLE>
+    : public ASplitter<DimConst<2>, InterpConst<2>, RefinedParticlesConst<8>>
+{
+    using Dispatcher = SplitParticleDispatcher<DimConst<2>, RefinedParticlesConst<8>, Shape::Cross>;
+
+public:
+    Splitter() { split(weights_, deltas_); }
+
+    template<typename Weights, typename Deltas>
+    static void split(Weights& weights, Deltas& deltas)
+    {
+        split(weights, weight_vals_, deltas, delta_vals_);
+    }
+    template<typename Weights, typename Deltas>
+    static void split(Weights& weights, float const weight_vals[], //
+                      Deltas& deltas, float const delta_vals[])
+    {
+        Dispatcher::set_weights(weights, weight_vals);
+        Dispatcher::set_deltas(deltas, delta_vals);
+    }
+
+protected:
+    constexpr static float delta_vals_[]  = {0.828428, 1.236701};
+    constexpr static float weight_vals_[] = {0.178624, 0.071376};
+};
+
+
+template</*default pattern*/>
+class Splitter<DimConst<2>, InterpConst<2>, RefinedParticlesConst<8>>
+    : public Splitter<DimConst<2>, InterpConst<2>, RefinedParticlesConst<8>, Color::PINK,
+                      Color::PURPLE>
+{
+};
+
+
+/**************************************************************************/
+template<>
+class Splitter<DimConst<2>, InterpConst<3>, RefinedParticlesConst<25>>
+    : public ASplitter<DimConst<2>, InterpConst<3>, RefinedParticlesConst<25>>
+{
+    using Dispatcher              = ExactSplitParticleDispatcher<DimConst<2>>;
+    static constexpr uint8_t root = 5;
```

Comment:
In fact, I agree wit this root value... and this one happens t be the exact number of refined particles for the exact splitting at 1 D for this same interp order... and the exact refined particle is thi root value at the power `dim`